scripts/generatePackageSwift.swift

```diff
@@ -160,48 +160,52 @@ private func dependency(url: String, version: String, branch: String?, path: Str
     return ".package(url: \"\(url)\", .exact(\"\(version)\"))"
 }
 
-func generateTargets(_ releasedSDKs: [String]) {
-    let targetsBeginning = """
-    targets: [
-        .target(
-            name: "AWSClientRuntime",
-            dependencies: [
-                .product(name: "ClientRuntime", package: "smithy-swift"),
-                .product(name: "AwsCommonRuntimeKit", package: "aws-crt-swift")
-            ],
-            path: "./AWSClientRuntime/Sources"
-        ),
-        .testTarget(
-            name: "AWSClientRuntimeTests",
-            dependencies: [
-                "AWSClientRuntime",
-                .product(name: "SmithyTestUtil", package: "smithy-swift"),
-                .product(name: "ClientRuntime", package: "smithy-swift")
-            ],
-            path: "./AWSClientRuntime/Tests"
-        ),
-"""
-    print(targetsBeginning)
+func generateServiceTargets(_ releasedSDKs: [String]) {
+    print("        // MARK: - Service Targets")
+    for sdk in releasedSDKs {
+        print(#"        .target(name: "\#(sdk)", dependencies: ["AWSClientRuntime"], path: "./Sources/Services/\#(sdk)"),"#)
+    }
+    print()
+    print("        // MARK: - Service Test Targets")
+    print("        // TODO: enable test targets for all services")
```

Comment:
Do we have an issue filed for this? Can we add a link to the issue in the comment here as well?

Reply:
https://github.com/awslabs/aws-sdk-swift/issues/814